### PR TITLE
new lifecycle doc and handling secrets update

### PR DESCRIPTION
**Pull request by dbradish-microsoft**
The purpose of this PR is as follows:

1. Publish new lifecycle article
2. Remove outdated cli-versioning-identifiers.md
3. Update sensitive information article
4. Remote support level information from Terminology article

### CODE REVIEW

**Comment by dbradish-microsoft** [comment]
Reviewed content with @mikefRobbins and @dcaro.  New PR created due to merge conflicts.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+|-|-|
+|3.8|October 2024 |
+|3.9|October 2025 |
+|3.10|October 2026 |
+|3.11|October 2027 |
+
+Azure CLI can only be supported on operating systems where the above versions of Python are supported.
+
+* **Windows:** The currently supported versions of Windows client and server meet Python version requirements.
+* **macOS:** The currently supported version of macOS 10.9 and above meet Python version requirements.
```
Comment (bebound) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:165]: macOS 12 is the oldest OS that is not EOL.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+* **Linux:**
+  * Each supported operating system has a lifecycle defined by its sponsor organization.
+  * Support is typically removed when an operating system goes out of mainline support, at which time we stop testing and supporting it.
+  * Here are the supported Linux operating systems for the Azure CLI:
+
+    |Operating system|Version|End of support|
+    |-|-|-|
+    | Ubuntu| 20.04 LTS| April 2025 |
+    | | 22.04 LTS | April 2027 |
```
Comment (bebound) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:174]: Ubuntu 24.04 is also supported.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    | Debian | 10 | June 2024 |
+    | | 11 | |
+    | | 12 | |
+    | RHEL | 8 | May 31, 2029 |
+    | | 9 | May 31, 2032 |
+    | CentOS Stream | 9 | Estimated 2027 |
+    | Mariner | 2.0 | |
+    | Alpine | 3.17 | November 22, 2024 |
+    | | 3.16 | May 23, 2024 |
```
Comment (bebound) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:183]: We don't support Alpine. It installs CLI from pip.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+| 2.64.0         | August 2024      |                  | Minor        | LTS          | 2.65.0                  |
+| 2.65.0         | September 2024   |                  | Minor        | LTS          | 2.66.0                  |
+| 2.66.0         | October 2024     |                  | Minor        | LTS          | _**2.79.0**_            |
+| 2.67.0         | November 2024    | Yes              | Major        | STS          | 2.68.0                  |
+| 2.68.0         | December 2024    |                  | Minor        | STS          | 2.69.0                  |
+| 2.69.0         | January 2025     |                  | Minor        | STS          | 2.70.0                  |
+| 2.70.0         | February 2025    |                  | Minor        | STS          | 2.71.0                  |
+| 2.71.0         | March 2025       |                  | Minor        | STS          | 2.72.0                  |
+| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
+| 2.72.0         | May 2025         | Yes              | Major        | LTS          | 2.73.0                  |
```
Comment (jiasli) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:123]:
```suggestion
| 2.73.0         | May 2025         | Yes              | Major        | LTS          | 2.91.0                  |
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+| 2.63.0         | July 2024        |                  | Minor        | LTS          | 2.64.0                  |
+| 2.64.0         | August 2024      |                  | Minor        | LTS          | 2.65.0                  |
+| 2.65.0         | September 2024   |                  | Minor        | LTS          | 2.66.0                  |
+| 2.66.0         | October 2024     |                  | Minor        | LTS          | _**2.79.0**_            |
+| 2.67.0         | November 2024    | Yes              | Major        | STS          | 2.68.0                  |
+| 2.68.0         | December 2024    |                  | Minor        | STS          | 2.69.0                  |
+| 2.69.0         | January 2025     |                  | Minor        | STS          | 2.70.0                  |
+| 2.70.0         | February 2025    |                  | Minor        | STS          | 2.71.0                  |
+| 2.71.0         | March 2025       |                  | Minor        | STS          | 2.72.0                  |
+| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
```
Comment (jiasli) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:122]: My understanding is that as an STS, 2.72.0 support is ended immediately after 2.73.0 is released.

```suggestion
| 2.72.0         | April 2025       |                  | Minor        | STS          | _**2.73.0**_            |
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+    | | 12 | |
+    | RHEL | 8 | May 31, 2029 |
+    | | 9 | May 31, 2032 |
+    | CentOS Stream | 9 | Estimated 2027 |
+    | Mariner | 2.0 | |
+    | Alpine | 3.17 | November 22, 2024 |
+    | | 3.16 | May 23, 2024 |
+
+    > [!NOTE]
+    > On RHEL 7, only Azure CLI 2.38 is supported and will receive security fixes until June 30, 2024.
```
Comment (bebound) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:186]: It is EOL now.

We also need to modify this statement in doc:
> Azure CLI [2.38.2](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#june-21-2024) (released on June 21, 2024) is the last version supported on RHEL 7 and Centos 7 and will continue to receive security fixes until the end of life of the operating systems. Microsoft isn't providing other updates or bug fixes for these distributions.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:165]: @bebound, `10.9` has been changed to `12` in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455). Please leave additional comments in the new PR.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 3 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:183]: @bebound, `Alpine` has been removed from the chart in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455). Please leave additional comments in the new PR.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 4 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:123]: @jiasli , I've made this change in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455), but I feel this needs more conversation. Please leave additional comments in the new PR.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 5 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:122]: @jiasli , I've made this change in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455), but I feel this needs more conversation. Please leave additional comments in the new PR.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 6 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:186]: @bebound, I've made this change in PR [#4455](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4455). Please leave additional comments in the new PR.

**Pull request merged**